TheMET/TheMET/Extentions/UISearchBar+apply.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UISearchBar+apply.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 03/08/2023.
+//
+
+import Foundation
+import UIKit
+
+extension UISearchBar {
+    
+    func apply(barTintColor: UIColor?,  textFieldBackgroundColor: UIColor?, textFieldColor: UIColor?) {
+        if let barTintColor = barTintColor,
+           let textFieldBackgroundColor = textFieldBackgroundColor,
+           let textFieldColor = textFieldColor,
+           let textField = self.value(forKey: "searchField") as? UITextField {
+            self.barTintColor = barTintColor
+            self.searchTextField.backgroundColor = textFieldBackgroundColor
+            self.searchTextField.attributedPlaceholder = NSAttributedString(string: NSLocalizedString("search", comment: ""), attributes: [NSAttributedString.Key.foregroundColor: textFieldColor])
+            textField.textColor = textFieldColor
+            let glassIconView = textField.leftView as! UIImageView
+            glassIconView.image = glassIconView.image?.withRenderingMode(UIImage.RenderingMode.alwaysTemplate)
+            glassIconView.tintColor = textFieldColor
+            let clearButton = textField.value(forKey: "clearButton") as! UIButton
```

Comment:
In this case there is no other way to configure the `clearButton` rather than get it out with `value(forKey:)`.
But be aware that Apple can change the private property name `clearButton` and this method will return `nil`. It will crash the App -> the proper unwrap is needed.